### ISSUE TITLE
add the NARPS dataset as a submodule

### ISSUE THREAD

**Issue by Remi-Gau** [Inria-Empenn/narps_open_pipelines#12]
**Is your feature request related to a problem? Please describe.**

Not really a problem but it may help standardize paths if we can always point to the data by using a path that is relative to the root of the repo.

**Describe the solution you'd like**

```bash
datalad install -d . \
                 -s https://github.com/OpenNeuroDatasets/ds001734.git \
                 data/original
```

**Comment by Remi-Gau** [comment]
This currently fails because `.gitmodules` is allegedly in the `.gitignore` (??? WTF ???)

And a `cat .gitignore | grep module` returns empty.

```bash
datalad install -d . -s https://github.com/OpenNeuroDatasets/ds001734.git data/original 

[INFO   ] scanning for annexed files (this may take some time)                                                                             
[INFO   ] Remote origin not usable by git-annex; setting annex-ignore                                                                      
[INFO   ] https://github.com/OpenNeuroDatasets/ds001734.git/config download failed: Not Found 
[INFO   ] access to 1 dataset sibling s3-PRIVATE not auto-enabled, enable with:
|               datalad siblings -d "/home/remi/github/narps_open_pipelines/data/original" enable -s s3-PRIVATE 
install(ok): data/original (dataset)
add(ok): data/original (file)                                                                                                              
[WARNING] Received an exception GitIgnoreError(The following paths are ignored by one of your .gitignore files:                            
.gitmodules
hint: Use -f if you really want to add them.
hint: Turn this message off by running
hint: "git config advice.addIgnoredFile false"
). Canceling not-yet running jobs and waiting for completion of running. You can force earlier forceful exit by Ctrl-C. 
[INFO   ] Canceled 0 out of 0 jobs. 0 left running.                                                                                        
Total:   0%|                                                                                            | 0.00/1.00 [00:00<?, ? datasets/s]The following paths are ignored by one of your .gitignore files:
.gitmodules
hint: Use -f if you really want to add them.
hint: Turn this message off by running
hint: "git config advice.addIgnoredFile false"

The following paths are ignored by one of your .gitignore files:
.gitmodules
hint: Use -f if you really want to add them.
hint: Turn this message off by running
hint: "git config advice.addIgnoredFile false"
```

**Comment by jadecci** [comment]
Looks like there's a `.*` in `.gitignore`

**Comment by Remi-Gau** [comment]
Awesome !!!
thanks for the catch!!
